darts/timeseries.py

```diff
@@ -541,7 +567,8 @@ def from_dataframe(df: pd.DataFrame,
     def from_times_and_values(times: pd.DatetimeIndex,
                               values: np.ndarray,
                               freq: Optional[str] = None,
-                              fill_missing_dates: Optional[bool] = True) -> 'TimeSeries':
+                              fill_missing_dates: Optional[bool] = True,
+                              column: Optional[str] = None,) -> 'TimeSeries':
```

Comment:
This setup with the `column` parameter assumes that we are creating a univariate time series, right? But technically we can also use this function to create a multivariate time series by passing a 2- 
dimensional numpy array as `values`. So we can just change the data type to also allow for a list of strings to support multivariate time series creation, right?

Reply:
you are right I would have to check that everything works with 2d array

Reply:
I think you'll have to use an argument similar to the `init` method: `columns: Optional[Union[List[str], str]]`.

Reply:
I changed it to rely more on pandas by accepting a `pd._typing.Axes` as columns parameter type. It's up to the user to come with either a list of columns or a pandas index...